content/2017/france/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "EclipseCon France 2017"
+date: 2017-06-21T08:00:00-04:00
+description: "Conference for the Eclipse Ecosystem | June 21 - 22, 2017 | Toulouse, France"
+page_favicon: "2017/france/favicon.ico"
+disable_css: "true"
```

Comment:
I believe the disable_css is preventing styles.css from being added to the page. We need this for this Font awesone icons in the footer.

Reply:
removed